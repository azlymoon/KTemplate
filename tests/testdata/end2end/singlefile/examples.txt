{% let $v = leaf %}
{% for $item in items %}
    {# comment #}
    {% let $s = $item ~ nested.leaf ~ $v %}
    {% if $item %}
        > {{ $s }}
    {% end %}
{% end %}
