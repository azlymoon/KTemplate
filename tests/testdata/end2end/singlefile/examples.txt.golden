

    
    
    
        > a2010
    

    
    
    
        > b2010
    

    
    
    
        > c2010
    

    
    
    

    
    
    
        > d2010
    

    
    
    
        > e2010
    

    
    
    
        > f2010
    

    
    
    

